### PR TITLE
switch swap endpoint to localhost

### PR DESCRIPTION
**Pull request by vandot**
This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/1731)

### CODE REVIEW

File: packaging/bee.yaml [hunk 1]
```diff
@@ -60,8 +60,8 @@ password-file: /var/lib/bee/password
 # standalone: false
 ## enable swap (default true)
 # swap-enable: true
-## swap ethereum blockchain endpoint (default "http://localhost:8545")
-swap-endpoint: https://rpc.slock.it/goerli
+## swap ethereum blockchain endpoint (default "ws://localhost:8546")
```
Comment (Eknir) [inline packaging/bee.yaml:63]: Why 8546 and not 8545?

File: .goreleaser.yml [hunk 2]
```diff
@@ -195,7 +195,8 @@ brews:
         Logs:   #{var}/log/swarm-bee/bee.log
         Config: #{etc}/swarm-bee/bee.yaml
 
-        Bee has SWAP enabled and by default is using slock.it goerli ethereum endpoint.
+        Bee has SWAP enabled and by default is using ws://localhost:8546 ethereum endpoint.
+        If needed obtain Infura account and set it.
```
Comment (Eknir) [inline .goreleaser.yml:199]: I liked your comment in `packaging/docker/README.md` (line 15).

File: packaging/deb/postinst [hunk 3]
```diff
@@ -11,7 +11,8 @@ if [ "$1" = "configure" ]; then
 Logs:   journalctl -f -u bee.service
 Config: /etc/bee/bee.yaml
 
-Bee has SWAP enabled and by default is using slock.it goerli ethereum endpoint.
+Bee has SWAP enabled and by default is using ws://localhost:8546 ethereum endpoint.
+If needed obtain Infura account and set it.
```
Comment (Eknir) [inline packaging/deb/postinst:15]: I liked your comment in `packaging/docker/README.md` (line 15). Could you update this line and give a bit more direction to the user (as in `packaging/docker`?

File: packaging/rpm/post [hunk 4]
```diff
@@ -10,7 +10,8 @@ if [ $1 -eq 1 ] ; then
 Logs:   journalctl -f -u bee.service
 Config: /etc/bee/bee.yaml
 
-Bee has SWAP enabled and by default is using slock.it goerli ethereum endpoint.
+Bee has SWAP enabled and by default is using ws://localhost:8546 ethereum endpoint.
+If needed obtain Infura account and set it.
```
Comment (Eknir) [inline packaging/rpm/post:14]: I liked your comment in `packaging/docker/README.md` (line 15). Could you update this line and give a bit more direction to the user (as in `packaging/docker`?

File: packaging/bee.yaml (hunk 1 above)
Comment (ldeffenb) [reply packaging/bee.yaml:63]: From https://geth.ethereum.org/docs/rpc/server:
HTTP Server: The default listening port is 8545.
WebSocket Server: The default WebSocket port is 8546.

File: .goreleaser.yml (hunk 2 above)
Comment (acud) [reply .goreleaser.yml:199]: I think we need to get rid of the SWAP construct from the CLI. Also, there is the storage incentives enabled by default and the backend is shared between them so we cannot and probably should not affiliate the ethereum backend just with a swap context

**Review by acud — APPROVED** [review]

File: .goreleaser.yml (hunk 2 above)
Comment (vandot) [reply .goreleaser.yml:199]: @acud what wording to use if not SWAP?

**Comment by vandot** [comment]
@Eknir is it better now?

File: .goreleaser.yml [hunk 5]
```diff
@@ -175,6 +175,7 @@ scoop:
     - "if(!(Test-Path $persist_dir\\password)){[System.Web.Security.Membership]::GeneratePassword(32,5) | Set-Content -Path $persist_dir\\password}"
     - "if(sc.exe query SwarmBeeSvc | Select-String FAILED){sc.exe create SwarmBeeSvc binPath= \"$dir\\bee.exe start --config=$persist_dir\\bee.yaml\" type= share start= auto displayName= \"Bee\"; sc.exe description SwarmBeeSvc \"Swarm client implemented in Go.\"}"
     - "if($global){$ETH_ADDRESS = (((C:\\ProgramData\\scoop\\shims\\bee.exe init --config=$persist_dir\\bee.yaml | Select-String ethereum) -split ' ‘)[-1]).Trim('\"')} else {$ETH_ADDRESS = (((bee.exe init --config=$persist_dir\\bee.yaml | Select-String ethereum) -split ' ‘)[-1]).Trim('\"')}; Write-Host \"Please make sure there is sufficient eth and bzz available on $ETH_ADDRESS address.\nLearn how to fund your node by visiting our docs at https://docs.ethswarm.org/docs/installation/fund-your-node\""
+    - "Write-Host \"Obtain free Infura account and set: swap-endpoint: wss://goerli.infura.io/ws/v3/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx inside $persist_dir\\bee.yaml\""
```
Comment (Eknir) [inline .goreleaser.yml:178]: Missing article. Obtain a free (...)

File: .goreleaser.yml [hunk 6]
```diff
@@ -195,7 +196,9 @@ brews:
         Logs:   #{var}/log/swarm-bee/bee.log
         Config: #{etc}/swarm-bee/bee.yaml
 
-        Bee has SWAP enabled and by default is using slock.it goerli ethereum endpoint.
+        Bee has SWAP enabled and by default is using ws://localhost:8546 ethereum endpoint.
+        If needed obtain free Infura account and set:
```
Comment (Eknir) [inline .goreleaser.yml:200]: Missing article

File: packaging/deb/postinst [hunk 7]
```diff
@@ -11,7 +11,9 @@ if [ "$1" = "configure" ]; then
 Logs:   journalctl -f -u bee.service
 Config: /etc/bee/bee.yaml
 
-Bee has SWAP enabled and by default is using slock.it goerli ethereum endpoint.
+Bee has SWAP enabled and by default is using ws://localhost:8546 ethereum endpoint.
+If needed obtain free Infura account and set:
```
Comment (Eknir) [inline packaging/deb/postinst:15]: article

**Review by Eknir — APPROVED** [review]
Once my last comment (see Mattermost) is taken into account, you can merge it! Thanks :)

**Comment by Eknir** [comment]
LGTM! Can you remove the draft status from this PR and/or merge?

**Pull request merged**